packages/ui-kit/src/components/DataGrid/DataGrid.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+
+import axiosInstance from '../../../../../app/storybook/src/axios'
+import { storybookCodeTemplate, useStorybookAccessToken } from '../../helpers'
+
+import { DataGridComponent, DataGrid as DataGridSource } from './DataGrid'
+
+const DataGrid = (args: Story['args']) => {
+  const { accessToken } = useStorybookAccessToken(axiosInstance)
+  const ref = React.useRef(null)
+
+  if (!accessToken && args?.query) {
+    return null
+  }
+
+  return (
+    <div style={{ maxHeight: '400px' }}>
+      <DataGridSource
+        ref={ref}
+        {...{
+          ...args,
+          query: args?.query
+            ? {
+                ...args?.query,
+                accessToken
+              }
+            : undefined
+        }}
+      />
+    </div>
+  )
+}
+
+const meta: Meta<typeof DataGridComponent> = {
+  title: 'Components/DataGrid',
+  component: DataGridComponent,
+  argTypes: {
+    baseTheme: {
+      table: {
+        disable: true
+      }
+    }
+  },
+  parameters: {
+    controls: { sort: 'alpha' },
+    codeTemplate: storybookCodeTemplate
+  }
+}
+
+type Story = StoryObj<typeof DataGridComponent>
+
+export default meta
+
+export const Basic: Story = {
+  args: {
+    query: {
+      dataPool: {
+        name: 'TacoSoft Demo Data'
```

Comment:
```suggestion
        name: `process.env.STORYBOOK_DATA_POOL_UNIQUE_NAME_1 ?? ''`
```
e.g. [SimpleFilter](https://github.com/propeldata/ui-kit/blob/main/packages/ui-kit/src/components/SimpleFilter/SimpleFilter.stories.tsx#L81)